src/Core/Config/Cache/ConfigCacheSaver.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+namespace Friendica\Core\Config\Cache;
+
+/**
+ * The ConfigCacheSaver saves specific variables back from the ConfigCache (@see ConfigCache )
+ * into the config-files
+ *
+ * It is capable of loading the following config files:
+ * - *.config.php   (current)
+ * - *.ini.php      (deprecated)
+ * - *.htconfig.php (deprecated)
+ */
+class ConfigCacheSaver extends ConfigCacheManager
+{
+	/**
+	 * The standard indentation for config files
+	 * @var string
+	 */
+	const INDENT = "\t";
+
+	/**
+	 * Saves a given value to the config file
+	 * Either it replaces the current value or it will get added
+	 *
+	 * @param string $cat   The configuration category
+	 * @param string $key   The configuration key
+	 * @param string $value The new value
+	 */
+	public function saveToConfigFile($cat, $key, $value)
```

Comment:
Didn't you say there could be multiple changes added to a file? Can you please make a structure where changes are added to a list and then committed in one go?